ts/ngx_core.d.ts

```diff
@@ -250,7 +250,7 @@ interface NgxFetchOptions {
 declare class SharedMemoryError extends Error {}
 
 type NgxSharedDictValue = string | number;
-type NgxKeyValuePair<V> = { key: string, value: V };
+type NgxKeyValuePair<V> = [key: string, value: V];
```

Comment:
Also, it seems here we can remove names `key` and `value`.

To `type NgxKeyValuePair<V> = [string, V];`